packages/lesswrong/server/repos/SubscriptionsRepo.ts

```diff
@@ -0,0 +1,37 @@
+import Subscriptions from '../../server/collections/subscriptions/collection';
+import AbstractRepo from './AbstractRepo';
+import { recordPerfMetrics } from './perfMetricWrapper';
+
+class SubscriptionsRepo extends AbstractRepo<"Subscriptions"> {
+  constructor() {
+    super(Subscriptions);
+  }
+
+  /**
+   * Fetches the IDs of users to whom the given user is subscribed.
+   */
+  async getSubscribedToUserIds(currentUserId: string): Promise<string[]> {
+    const results = await this.getRawDb().manyOrNone<{
+      userId: string;
+    }>(
+      `
+      -- SubscriptionsRepo.getSubscribedToUserIds
```

Comment:
I don't think this needs to be a sql query written out by hand?  Can't this just be `Subscriptions.find({ state: 'subscribed', deleted: { $ne: true }, collectionName: 'Users', type: { $in: [...] }, userId: '...' })`?